### PR TITLE
Issue #93 fixed

### LINKED ISSUE #93: Implementing Admin Dashboard navbar

### PR DESCRIPTION
**Pull request by I3han**
Implementing Admin Dashboard navbar

### CODE REVIEW

File: src/layouts/DashboardLayout.jsx [hunk 1]
```diff
@@ -11,6 +11,7 @@ const DashboardLayout = (ViewComponent) => {
           {/* <Import the nav bar> */}
           <Navbar />
           <AdminNavBar />
+
```
Comment (I3han) [inline src/layouts/DashboardLayout.jsx]: .

File: src/layouts/DashboardLayout.jsx [hunk 2]
```diff
@@ -9,9 +9,9 @@ const DashboardLayout = (ViewComponent) => {
       return (
         <>
           {/* <Import the nav bar> */}
-          <Navbar />
```
Comment (I3han) [inline src/layouts/DashboardLayout.jsx:11]: Navbar removed

File: src/layouts/DashboardLayout.jsx [hunk 3]
```diff
@@ -9,9 +9,9 @@ const DashboardLayout = (ViewComponent) => {
...
+
           <AdminNavBar />
-
```
Comment (I3han) [inline src/layouts/DashboardLayout.jsx]: .

**Review by I3han — COMMENTED** [review]
checked

**Comment by kelumsampath** [comment]
@I3han , make sure to pass all the checks in build. 
Capture
seems @fortawesome/free-solid-svg-icons npm not installed properly. Install that npm as dev dependency  (npm i --save @fortawesome/free-solid-svg-icons) . then it will update your package.json .

**Review by dulangaheshan — APPROVED** [review]

**Review by ChanakaUOMIT — APPROVED** [review]

**Pull request merged**